app/models/chargeback.rb

```diff
@@ -146,11 +144,9 @@ def get_rates(perf)
       end
   end
 
-  def self.calculate_costs(metric_rollup_records, rates, hours_in_interval)
-    calculated_costs = {}
-
-    chargeback_fields_present                = metric_rollup_records.count(&:chargeback_fields_present?)
-    calculated_costs['fixed_compute_metric'] = chargeback_fields_present if chargeback_fields_present
+  def calculate_costs(metric_rollup_records, rates, hours_in_interval)
+    chargeback_fields_present = metric_rollup_records.count(&:chargeback_fields_present?)
+    self.fixed_compute_metric = chargeback_fields_present if chargeback_fields_present
```

Comment:
This feels a bit weird or fragile because `fixed_compute_metric` is only defined in the descendant classes and in in the base class.